tests/test-buffer.js

```diff
@@ -9,14 +9,28 @@ var buff;
 var lens = [[-1, 0],
             [0, 0],
             [8, 8],
-            [8.5, 8],
-            [4294967295, 4294967295],
-            [4294967296, 4294967295]];
-for(var i = 0; i < lens.length; i++) {
+            [8.5, 8]];
+for (var i = 0; i < lens.length; i++) {
     buff = new Buffer(lens[i][0]);
     assert(buff.length === lens[i][1],
-           "The length of Buffer(" + lens[i][0] + ") expected:" + lens[i][1] +
-           " got:" + buff.length);
+           "Creating buffer of length " + lens[i][0] + " expected:" +
+           lens[i][1] + " got:" + buff.length);
+}
+
+// thrown errors are allowed on these lengths
+lens = [[4294967295, 4294967295],
+        [4294967296, 4294967295]];
```

Comment:
This wouldn't be bad, but it's kind of irrelevant for these small devices we're really targeting. They will always run out of RAM way before this "max length". I guess I can add it under a Linux ifdef so it doesn't add more code to the small device case.